.github/workflows/deploy-prep.yml

```
@@ -0,0 +1,739 @@
+          metadata:
+            name: resumesync-web
+          spec:
+            replicas: $([ "${ENVIRONMENT}" = "production" ] && echo "2" || echo "1")
+          EOF
+
+      - name: Build and validate manifests
+        env:
+          ENVIRONMENT: ${{ needs.prepare-deployment.outputs.environment }}
+        run: |
+          echo "Building Kubernetes manifests for ${ENVIRONMENT}..."
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)